### ISSUE TITLE
Incompatible with Chefdk 0.4+

### ISSUE THREAD

**Issue by donaldguy** [chef-cookbooks/libarchive#9]
Running the chef-dk installer results in an embedded ruby install which includes version 0.0.6 of the gem `archive`

This is preferred in the `require 'archive'` to `libarchive-ruby`, resulting in failed chef runs with output similar to 

```
NameError
---------
uninitialized constant Archive::EXTRACT_OWNER

Cookbook Trace:
---------------
/chef/cookbooks/libarchive/libraries/helper.rb:14:in `extract_option_map'
/chef/cookbooks/libarchive/libraries/helper.rb:34:in `block in extract'
/chef/cookbooks/libarchive/libraries/helper.rb:34:in `collect!'
/chef/cookbooks/libarchive/libraries/helper.rb:34:in `extract'
/chef/cookbooks/libarchive/providers/file.rb:18:in `block in class_from_file'
```

**Comment by johnbellone** [comment]
@donaldguy I think this may be fixed. Could you try again?

**Comment by iennae** [comment]
Hi,

This has been fixed since chef-client 12.1. Thanks!
